milestone2.ts

```diff
@@ -164,9 +171,12 @@ let getVolunteerInfo: GetVolunteerInfo = (vol) => {
 /* these criteria, return undefined
  */
 
-let daBigTest: GetVolunteer; // Code here
+let daBigTest: GetVolunteer = (data) => {
+  let person = data.find(volunteer => volunteer.city === "SLO" && volunteer.age > 40);
+  return (typeof person === 'undefined' ? person : updateVolunteer(person, {position: "staff"}));
```

Comment:
the typeof here is unnecessary, the same comparison can be done if you do person === undefined. Great use of the ternary operator though!